### ISSUE TITLE
Feature Request: Bruteforces Table Lacks Primary Key, Causing Issues with MySQL Group Replication

### ISSUE THREAD

**Issue by JotaPePG** [MISP/MISP#10074]
### Is your feature request related to a problem? Please describe.

Yes, the feature request is related to a problem when running MISP in a MySQL cluster with Group Replication. The bruteforces table is created without a primary key, which violates the requirements of Group Replication.

Without a primary key, the replication process fails, causing MISP to stop working in this clustered setup. This makes it difficult to deploy MISP in environments that rely on MySQL Group Replication for high availability and fault tolerance.

### Describe the solution you'd like

The solution is to modify the schema of the bruteforces table to include a primary key.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by iglocska** [comment]
Yeah, we had the same issue on our instances and just added an auto incrementing primary key. It absolutely would make sense to make this baseline in the next release.

**Comment by JotaPePG** [comment]
It persists in v2.5.6
